pkg/kapp/resources/resources.go

```diff
@@ -362,7 +362,7 @@ func (c *ResourcesImpl) Get(resource Resource) (Resource, error) {
 	return NewResourceUnstructured(*item, resType), nil
 }
 
-func (c *ResourcesImpl) Exists(resource Resource) (bool, error) {
+func (c *ResourcesImpl) Exists(resource Resource, existsOpts ...string) (bool, error) {
```

Comment:
Should we be using a simple boolean instead of allowing any number of string arguments to be passed to the `Exists()` function? That would lead to a lot of refactoring but I think it makes it more robust? Maybe create a struct like `ExistsOpts` if we wanna accommodate more options in the future?

Reply:
Changed to have a struct instead of string variadac.